README.md

```diff
@@ -81,9 +81,20 @@ export default {
 };
 ```
 
+`AWS_SECRETS_MANAGER_NAME` can also be a list of strings that each correspond to a separate AWS Secret Manager bucket name. Each bucket from the list is evaluated in order. In the scenario that a specific secret appears in two buckets the value will be overwritten.
+
+```ts
+// config.default.ts
+export default {
+  AWS_SECRETS_MANAGER_NAME: ['production/myapp/config', 'production/myapp/another-config'],
```

Comment:
I'm not sure if this should be an actual array. When read from an environment variable everything is a string so maybe this should just be `production/myapp/config, production/myapp/another-config`